src/wasm/wasm-binary.cpp

```diff
@@ -2208,6 +2212,20 @@ BinaryConsts::ASTNodes WasmBinaryBuilder::readExpression(Expression*& curr) {
       visitCallIndirect(
         (curr = allocator.alloc<CallIndirect>())->cast<CallIndirect>());
       break;
+    case BinaryConsts::RetCallFunction: {
+      auto call = allocator.alloc<Call>();
+      call->isReturn = true;
```

Comment:
This PR doesn't set isReturn (to false) for normal calls, I think, leaving it an uninitialized value?

Would be nice to refactor so the 4 cases for calls are more similar.